### PR TITLE
I think I got the basics done

### PR DESCRIPTION
**Pull request by natasha-tullos**

### CODE REVIEW

**Comment by natasha-tullos** [comment]
Uhhh so there's a readme conflict going on

**Comment by natasha-tullos** [comment]
I did it though?? 😂

**Comment by jhnsnc** [comment]
I like that you separated the routes functions into 2 files. It makes your `server.js` really nice to read.

Make sure you're consistent about if you're sending back JSON (`res.json(data);`) vs sending back plaintext/untyped text (`res.send(data);`).

The models themselves are very neatly done. Did you find that they were easy to work with when you tested your API?

Nice!

File: models/index.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 require('dotenv').config();
 // TODO: include all model files here (and export models together below)
-const testModels = require('./test');
+
+const projectModel = require(`./project.js`);
+const developerModel = require(`./developer.js`);
 
 // connect to Mongo DB
-mongoose.connection.openUri(process.env.MONGODB_URI || process.env.DB_CONN, {}, function(err, conn) {
+mongoose.connection.openUri(process.env.MONGODB_URI || "https://sheltered-taiga-43222.herokuapp.com/", {}, function(err, conn) {
```
Comment (jhnsnc) [inline models/index.js:9]: This probably isn't what you intended to do. This function expects the location of your Mongo DB, so pointing it to a Node server on Heroku probably won't work as intended.

Instead, you could give it a different environment variable to fall back on.

**Pull request closed without merging**